libs/utils/locales.js

```diff
@@ -3,8 +3,7 @@ export default {
   ae_ar: { ietf: 'ar-AE', tk: 'lpk1hwn.css', dir: 'rtl' },
   ae_en: { ietf: 'en', tk: 'hah7vzn.css' },
   africa: { ietf: 'en', tk: 'hah7vzn.css' },
-  ar: { ietf: 'ar', tk: 'lpk1hwn.css', dir: 'rtl' },
-  ar_es: { ietf: 'es-AR', tk: 'hah7vzn.css' },
+  ar: { ietf: 'es-AR', tk: 'oln4yqj.css' },
```

Comment:
Provided a comment in the story, I'd stick with `hah7vzn` for consistency purposes, since other Spanish locales are using the same one.